packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/template.html

```diff
@@ -0,0 +1,349 @@
+                    ></span>
+                    <span
+                        class="font-weight-bold"
+                        ng-bind="$ctrl.initalConfig.redundancyFactor"
+                    ></span>
+                    <span
+                        class="oui-icon oui-icon-info ml-1"
+                        data-oui-popover="{{:: 'nutanix_redeploy_initial_configuration_replication_description' | translate}}"
+                        data-oui-popover-placement="right"
+                    ></span>
+                </li>
+                <li class="align-bottom">
+                    <span
+                        data-translate="nutanix_redeploy_initial_configuration_erasure_coding"
+                    ></span>
+                    <span
+                        class="font-weight-bold"
+                        ng-bind="$ctrl.initalConfig.erasureCoding ? 'nutanix_redeploy_initial_configuration_erasure_coding_enabled' : 'nutanix_redeploy_initial_configuration_erasure_coding_disabled' | translate"
+                    ></span>
+                    <span
+                        class="oui-icon oui-icon-info ml-1"
+                        data-oui-popover="{{:: 'nutanix_redeploy_initial_configuration_erasure_coding_description' | translate}}"
+                        data-oui-popover-placement="right"
+                    ></span>
+                </li>
+                <li class="align-bottom">
+                    <span
+                        data-translate="nutanix_redeploy_initial_configuration_prism_element_aos"
+                    ></span>
+                    <span
+                        class="font-weight-bold"
+                        ng-bind="$ctrl.initalConfig.version"
+                    ></span>
+                    <span
+                        class="oui-icon oui-icon-info ml-1"
+                        data-oui-popover="{{:: 'nutanix_redeploy_initial_configuration_prism_element_aos_description' | translate}}"
+                        data-oui-popover-placement="right"
+                    ></span>
+                </li>
+            </ul>
+        </oui-step-form>
+
+        <oui-step-form
+            data-ng-if="$ctrl.redeployMethod === $ctrl.REDEPLOY_CUSTOM_CONFIG"
+            data-name="customConfig"
+            data-header="{{:: 'nutanix_redeploy_step2_title_custom_configuration' | translate}}"
+            data-position="1"
+            data-submit-text="{{:: 'nutanix_redeploy_button' | translate}}"
+            data-prevent-next="true"
+        >
+            <div class="mb-3">
+                <h5
+                    data-translate="nutanix_redeploy_custom_configuration_replication"
+                ></h5>
+                <p
+                    data-translate="nutanix_redeploy_custom_configuration_replication_description"
+                ></p>
+                <oui-select-picker
+                    data-ng-if="$ctrl.replicationFactors.rf2"
```

Comment:
this should be dynamic. There can be  1, 2 or more than RF's. Use ng-repeat and dynamically display all available RF's.